__tests__/app.test.js

```diff
@@ -32,3 +34,22 @@ describe("Generic handler", () => {
       });
   });
 });
+
+describe("/api", () => {
+  test("GET: 200 sends an object with all available endpoints.", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(typeof response.body).toBe("object");
+      });
+  });
+  test("Should check that all available APIs are included.", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.endpoints).toEqual(endpoints);
```

Comment:
Great use of the endpoints.json here to ensure the response looks exactly as expected!